src/com/mishiranu/dashchan/ui/posting/dialog/ReencodingDialog.java

```diff
@@ -115,5 +130,6 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
 			}
 		}
 		qualityLayoutHolder.setEnabled(allowQuality);
+		jpegToJpg.setEnabled(checkedId == IDS[JPEG_POSITION]);
```

Comment:
Hide jpeg related option if selected format is not jpeg.
```suggestion
		jpegToJpg.setVisibility(checkedId == IDS[JPEG_POSITION] ? View.VISIBLE : View.GONE);
```

Reply:
JPEG is the default option for the reencode feature, so the checkbox will always appear once. I don’t think that hiding it makes sense, in theory it will change the size of the interface + in the settings, Enable/Disable the element is usually used, not visibility